### DISCUSSION TITLE
Is there still support for Yocto SDK?

### DISCUSSION THREAD

**Discussion by gregwillits** [eclipse-embed-cdt/eclipse-plugins#599] [General Discussions]
I've been developing apps with standard CDT for desktop command line which are ultimately headed for an embedded Yocto project. Based on articles I read quite some time ago, the plan was to eventually move to Embedded CDT with the Yocto SDK.

However, revisiting those articles I am realizing they're quite old now, and suspiciously, there's no newer ones that I can find.

It looked to me before that Embedded CDT fully embraced Yocto SDK (or the other way around, the SDK had CDT integrations) — is that still true, or has that pairing gone stale?

Thanks for any info.

**Comment by TommyMurphyTM1234** [comment]
> Based on articles I read quite some time ago, the plan was to eventually move to Embedded CDT with the Yocto SDK.

It might help to link to these articles?

What is the nature of the Yocto support/integration that you're looking for?

**Comment by gregwillits** [comment]
@TommyMurphyTM1234 — I was hoping to be able to at least do cross-compile development of application(s) for a yocto-based OS. At the moment I'm having the OS itself, and mods I need to it, created by a 3rd party. It'll be quite some time before I can focus on understanding yocto layer & recipe development itself.

Articles like the following (and others). I just don't see anything newer which hinted at it not being a "thing" anymore.

https://community.nxp.com/t5/i-MX-Processors-Knowledge-Base/Setting-up-the-Eclipse-IDE-for-Yocto-Application-Development/ta-p/1110480

https://variwiki.com/index.php?title=Yocto_Programming_with_Eclipse_v2&release=mx6ul-yocto-kirkstone-5.15.71_2.2.0-v1.0

Meanwhile, I've discovered the following which clarifies that it has been a dead end for a while now. Bummer.

https://github.com/crops/eclipse-yocto

https://stackoverflow.com/questions/67018846/why-has-eclipse-yocto-plugin-support-been-abandoned

Anyway, the above answers my question. (Apparently stuck using VSCode.)

Thanks for your time.
